CONTRIBUTING.md

```diff
@@ -22,6 +22,19 @@ We welcome pull requests! Please include the following in the description:
 Please be aware that your pull request should pass existing tests and linting, and also contain corresponding tests
 for the feature or fix added.
 
+## Coding standard
+`go fmt` is used to format the code.
+
+You can use `task lint`, which will run linting using `golangci-lint`.
+
```

Comment:
Perhaps one word about lint version being compatible with the one potentially used at build?